tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    public function testRateLimit()
+    {
+        try {
+            for ($x = 0; $x <= 100; $x++) {
+                Customer::create();
```

Comment:
This is going to make our tests take ~forever (specifically adding 30s to each of our 12 builds). Can we do something else? We have the ability to mock HTTP responses now.